mlos_bench/mlos_bench/schedulers/sync_scheduler.py

```diff
@@ -15,24 +15,10 @@
 class SyncScheduler(Scheduler):
     """A simple single-threaded synchronous optimization loop implementation."""
 
-    def start(self) -> None:
```

Comment:
Moved to base class as it was nearly identical to what the ParallalScheduler would use.